src/test/java/space/obminyashka/items_exchange/controller/CategoryControllerIntegrationTest.java

```diff
@@ -158,6 +161,31 @@ void createCategory_whenCategoryNameHasDuplicate_shouldReturnBadRequest()
                 .andExpect(status().isBadRequest());
     }
 
+    @Test
+    @WithMockUser(username = USERNAME_ADMIN, roles = {ROLE_ADMIN})
+    @DataSet("database_init.yml")
+    void createCategory_whenCategoryNameHasInvalidSize_shouldReturnBadRequest()
+            throws Exception {
+
+        CategoryDto nonExistCategoryDtoWithInvalidName = createNonExistCategoryDtoWithInvalidName();
+
+        final var validationMessage = MessageSourceUtil.getMessageSource("invalid.size")
```

Comment:
You have the same logic into **updateCategory_whenCategoryNameHasInvalidSize_shouldReturnBadRequest** test as well. Would you mind extracting it into a new method?

Reply:
ok, i`ll extract